elements/analysis/fromipsumdump.cc

```diff
@@ -101,10 +101,10 @@ FromIPSummaryDump::configure(Vector<String> &conf, ErrorHandler *errh)
     .complete() < 0)
     return -1;
     if (_sampling_prob > (1 << SAMPLING_SHIFT)) {
-    errh->warning("SAMPLE probability reduced to 1");
-    _sampling_prob = (1 << SAMPLING_SHIFT);
+		errh->warning("SAMPLE probability reduced to 1");
```

Comment:
Make sure that the source code is space-separated with tab size = 4

Reply:
Should not have tabs at all... I'll add a style commit.

Reply:
Yes I know, when using space separation a tab consists of consecutive spaces